GNUmakefile

```diff
@@ -58,19 +58,23 @@ SHELL-NAME := makes go-yaml
 include $(MAKES)/clean.mk
 include $(MAKES)/shell.mk
 
-MAKES-CLEAN := $(dir $(YTS-DIR)) $(GOLANGCI-LINT)
+MAKES-CLEAN += $(dir $(YTS-DIR)) $(GOLANGCI-LINT)
 
 v ?=
 count ?= 1
 
 
 # Test rules:
-test: $(GO-DEPS)
-	go test$(if $v, -v) -vet=off ./...
+check:
+	$(MAKE) fmt
+	$(MAKE) tidy
+	$(MAKE) lint
+	$(MAKE) test
 
-test-data: $(YTS-DIR)
+test: test-unit test-yts-all
 
-test-all: test test-yts-all
+test-unit: $(GO-DEPS)
+	go test$(if $v, -v) -vet=off
```

Comment:
if you don't want to use ./...

Then we will have to consider a dedicated module for the CLI and yts, this way go test ./... will work as you expect, but also as any Go developper would expect to launch tests on the project

Reply:
I opened 

- https://github.com/yaml/go-yaml/pull/172
- https://github.com/yaml/go-yaml/pull/173

What do you think to be the best @SVilgelm ?

I prefer #172 as it doesn't hide the test suite locally.

Reply:
@SVilgelm and I paired on this for a minute and modified this PR minimally. 

This PR is really just about getting `make test` to work from scratch again.

Reply:
`go test internal/...` fails for me.
There's no testing of the internals in internals, just the testing framework tests.
We can address those later.
I want to get the makefile working again asap.